src/fastoad/models/performances/mission/segments/ground_speed_change.py

```diff
@@ -0,0 +1,107 @@
+"""Classes for acceleration/deceleration segments."""
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2021 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import List
+
+from scipy.constants import g
+
+from fastoad.model_base import FlightPoint
+from .base import GroundSegment
+from ..exceptions import FastFlightSegmentIncompleteFlightPoint
+
+_LOGGER = logging.getLogger(__name__)  # Logger for this module
+
+
+@dataclass
+class GroundSpeedChangeSegment(GroundSegment, mission_file_keyword="ground_speed_change"):
+    """
+    Computes a flight path segment where aircraft is accelerated or de-accelerated on the ground
+
+    The target must define an airspeed (equivalent, true or Mach) value.
+    """
+
```

Comment:
Here is the first of the 2 modifications that would use the suggested change in [`GroundEffectRaymer`](https://github.com/fast-aircraft-design/FAST-OAD/pull/481/files#diff-86d2c880fe0a8ceb8ce6e2663eba2fefec24a08ab5a207d69cdd3b25deaba50e).
```suggestion
    def compute_from_start_to_target(self, start: FlightPoint, target: FlightPoint) -> pd.DataFrame:
        self.polar_modifier.ground_altitude = start.altitude
        return super().compute_from_start_to_target(start, target)

```